arch/arm/src/Makefile

```diff
@@ -133,7 +133,7 @@ else
 endif
 
 ARCHSCRIPT := $(call CONVERT_PATH,$(ARCHSCRIPT))
-LDFLAGS += $(addprefix $(SCRIPT_OPT),$(addsuffix .tmp,$(ARCHSCRIPT))) $(EXTRALINKCMDS)
+LDFLAGS += $(addprefix $(SCRIPT_OPT) ,$(addsuffix .tmp,$(ARCHSCRIPT))) $(EXTRALINKCMDS)
```

Comment:
why change

Reply:
> why change

this is using to support "[support inputting multiple ldscript files when linking the nuttx.bin](https://github.com/apache/nuttx/pull/12884/commits/85ce364d4c60a9fd0729564b807788e7c199d8fc)"